From d540e40f7fc2af8a45ba86596a1856166c4e06f1 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 18 Jan 2013 14:12:56 +0000
Subject: [PATCH 0390/1077] LogService.exe massaisnmp.dll V1.0.4.1 Mantis
 0025600: MR SankyoICM300: DeviceHandler for MCR added GmrDev_AccessIS6x0
 added GmrDev_is49x0 added GmrDev_SankyoICM300

git-svn-id: svn://localhost/SelfServiceCommon/trunk@466 90b65887-3827-0410-9a23-83215b262276
---
 .../compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.cfg | 5 +++++
 .../compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.csp | 1 +
 .../GmrDev_SankyoICM300.GENERAL.compDefinition.cfg           | 3 +++
 .../GmrDev_SankyoICM300.GENERAL.compDefinition.csp           | 1 +
 .../compSource/GmrDev_is49x0.GENERAL.compDefinition.cfg      | 3 +++
 .../compSource/GmrDev_is49x0.GENERAL.compDefinition.csp      | 1 +
 6 files changed, 14 insertions(+)
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.csp
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.csp
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.cfg
 create mode 100644 SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.csp

diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.cfg
new file mode 100644
index 00000000..9137d7c5
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.cfg
@@ -0,0 +1,5 @@
+MPR_ACCESSIS6X0_READER;OK=8600;ERROR=8601 | 8602 | 8603 | 8604 | 8605 | 8606 | 8612
+MPR_ACCESSIS6X0_COMP;OK=MPR_ACCESSIS6X0_READER.OK
+CUSS_PPR_COMP(HIDDEN);OK=MPR_ACCESSIS6X0_COMP.OK
+CUSS_IMG_COMP(HIDDEN);OK=MPR_ACCESSIS6X0_COMP.OK
+CUSS_OCR_COMP(HIDDEN);OK=MPR_ACCESSIS6X0_COMP.OK
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.csp
new file mode 100644
index 00000000..a0f3019e
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_AccessIS6x0.GENERAL.compDefinition.csp
@@ -0,0 +1 @@
+HARDWARE_OK;&;MPR_ACCESSIS6X0_COMP.OK;GmrDev_AccessIS6x0.GENERAL.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.cfg
new file mode 100644
index 00000000..08e43ab3
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.cfg
@@ -0,0 +1,3 @@
+MPR_SANKYOICM300_READER;OK=8800;ERROR=8801 | 8802 | 8803 | 8804 | 8805 | 8806 | 8812
+MPR_SANKYOICM300_COMP;OK=MPR_SANKYOICM300_READER.OK
+CUSS_MCR_COMP(HIDDEN);OK=MPR_SANKYOICM300_COMP.OK
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.csp
new file mode 100644
index 00000000..a07dffcf
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_SankyoICM300.GENERAL.compDefinition.csp
@@ -0,0 +1 @@
+HARDWARE_OK;&;MPR_SANKYOICM300_COMP.OK;GmrDev_SankyoICM300.GENERAL.compDefinition
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.cfg b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.cfg
new file mode 100644
index 00000000..a405b25d
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.cfg
@@ -0,0 +1,3 @@
+MPR_IS49X0_READER;OK=8700;ERROR=8701 | 8702 | 8703 | 8704 | 8705 | 8706 | 8712
+MPR_IS49X0_COMP;OK=MPR_IS49X0_READER.OK
+CUSS_BCR_COMP(HIDDEN);OK=MPR_IS49X0_COMP.OK
\ No newline at end of file
diff --git a/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.csp b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.csp
new file mode 100644
index 00000000..f520f810
--- /dev/null
+++ b/SelfServiceCommon/Massai/cpp/LogService/compSource/GmrDev_is49x0.GENERAL.compDefinition.csp
@@ -0,0 +1 @@
+HARDWARE_OK;&;MPR_IS49X0_COMP.OK;GmrDev_is49x0.GENERAL.compDefinition
\ No newline at end of file
-- 
2.41.0.windows.1

